client/src/main/java/org/devfleet/esi/impl/service/ESICorporationServiceImpl.java

```diff
@@ -10,68 +14,73 @@
 
 import java.io.IOException;
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
+import java.util.Optional;
 
-class CorporationAPIImpl {
+import static java.util.Collections.emptyList;
```

Comment:
Fair enough. I'm used to them and don't see much of a problem with them, but they can take some time getting used to. 

Will revert to "normal" imports